consent/helpers.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+"""
+Helper functions for the Consent application.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+from django.contrib.auth.models import User
+
+# Will be using internal models at a later time.
+from enterprise.models import (
+    EnterpriseCourseEnrollment,
+    EnterpriseCustomer,
+    EnterpriseCustomerUser,
+    UserDataSharingConsentAudit,
+)
+
+
+def consent_exists(user_id, course_id, enterprise_customer_uuid):
+    """
+    Get whether any consent is associated with an EnterpriseCustomer.
+
+    :param user_id: The user that grants consent.
+    :param course_id: The course for which consent is granted.
+    :param enterprise_customer_uuid: The consent requester.
+    :return: Whether any consent (provided or unprovided) exists.
+    """
+    enterprise_customer = get_enterprise_customer(enterprise_customer_uuid)
+    enterprise_course_enrollment = get_enterprise_course_enrollment(user_id, course_id, enterprise_customer)
+    return bool(enterprise_course_enrollment or get_user_dsc_audit(user_id, enterprise_customer))
+
+
+def consent_provided(user_id, course_id, enterprise_customer_uuid):
+    """
+    Get whether consent is provided by the user to the Enterprise customer.
+
+    :param user_id: The user that grants consent.
+    :param course_id: The course for which consent is granted.
+    :param enterprise_customer_uuid: The consent requester.
+    :return: Whether consent is provided to the Enterprise customer by the user for a course.
+    """
+    enterprise_customer = get_enterprise_customer(enterprise_customer_uuid)
+    enterprise_course_enrollment = get_enterprise_course_enrollment(user_id, course_id, enterprise_customer)
+    provided = enterprise_course_enrollment is not None and enterprise_course_enrollment.consent_available
```

Comment:
nit: why not just `enterprise_course_enrollment and enterprise_course_enrollment.consent_available`?